mutiny-core/src/nodemanager.rs

```diff
@@ -1109,7 +1221,9 @@ impl NodeManager {
                 let msats = amount_sats * 1000;
                 let invoice = self.lnurl_client.get_invoice(&pay, msats).await?;
 
-                self.pay_invoice(from_node, &invoice.invoice(), None).await
+                let label = "LNURL Pay".to_string();
```

Comment:
Might want to expose `label` for `lnurl_pay` instead so we label the invoice like any other.